test/workers/test_websocketIOClientWorker.py

```diff
@@ -0,0 +1,42 @@
+import time
+
+import socketio
+import torch
+
+from syft.workers import WebsocketIOServerWorker, WebsocketIOClientWorker
+
+
+def create_dummy_client():
```

Comment:
Maybe move this to conftest?

Reply:
The dummy client must connect after the server has been created

Reply:
Cool, sgtm.